src/components/modal/modal.css

```diff
@@ -25,6 +25,10 @@
   outline: 0;
 }
 
+.scroll {
+  overflow: scroll;
```

Comment:
This will _always_ show the scrollbar regardless if it's needed or not. `auto` will only show it if necessary.

Reply:
Fair, let me get that fixed up.

Reply:
So I think we won't even need the `scrollable` prop: we just always have `overflow: auto`.